isis/src/base/objs/UserInterface/UserInterface.cpp

```diff
@@ -405,22 +405,22 @@ namespace Isis {
    */
   void UserInterface::loadCommandLine(QVector<QString> &args, bool ignoreAppName) {
     char **c_args;
-    
+
     if (ignoreAppName) {
       args.prepend("someapp");
     }
 
     c_args = (char**)malloc(sizeof(char*)*args.size());
-    
-    for (size_t i = 0; i < args.size(); i++) {
+
+    for (int i = 0; i < args.size(); i++) {
```

Comment:
What's the rationale for changing this to int?

Reply:
I was getting some warnings about it not being an int during the build.